src/condor_utils/file_transfer.h

```diff
@@ -115,10 +115,42 @@ class DataReuseDirectory;
 }
 
 
+// `TransferType` is currently being used by should be `TransferDirection`.
+enum class TransferClass {
+    none = 0,
+	input = 1,
+	output = 2,
+	checkpoint = 3,
```

Comment:
Wonky whitespacing... If we find other reasons for changes we may want to update this